packages/x-components/src/x-modules/experience-controls/store/types.ts

```diff
@@ -0,0 +1,62 @@
+import { XEventsTypes } from '../../../wiring';
+import { XStoreModule } from '../../../store/store.types';
+import { XActionContext } from '../../../store';
+/**
+ * Experience Controls store state.
+ *
+ * @public
+ */
+export interface ExperienceControlsState {
+  /**
+   * Configuration for the `ExperienceControls` module.
+   */
+  controls: { [key: string]: unknown };
+  events: Partial<XEventsTypes>;
+}
+
+/**
+ * Experience Controls store getters.
+ *
+ * @public
+ */
+export interface ExperienceControlsGetters {}
+
+/**
+ * Experience Controls store mutations.
+ *
+ * @public
+ */
+export interface ExperienceControlsMutations {
+  /**
+   * Sets the {@link ExperienceControlsState.controls} property.
+   *
+   * @param experienceControlsConfig - The new {@link ExperienceControlsState.config}.
```

Comment:
The `@param` in the doc of the mutation and the method signature have to have the same name.  

```suggestion
   * @param controls - The new {@link ExperienceControlsState.config}.
```